### PR TITLE
Optimization

### PR DESCRIPTION
**Pull request by bertho-zero**
Do not bind parameter in render

### CODE REVIEW

**Comment by bertho-zero** [comment]
Fixs https://github.com/react-component/collapse/issues/94

File: src/Collapse.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ function toArray(activeKey) {
   return currentActiveKey;
 }
 
-class Collapse extends Component {
+class Collapse extends PureComponent {
```
Comment (afc163) [inline src/Collapse.jsx]: Don't use PureComponent, we need to keep compatibility in React 15.

Comment (bertho-zero) [reply src/Collapse.jsx]: 15.3 is not enough? ant-design require version 16 or higher.

Comment (afc163) [reply src/Collapse.jsx]: ant-design don't require version 16 or higher, we still have test cases for React 15.

Comment (bertho-zero) [reply src/Collapse.jsx]: I saw that:
https://github.com/ant-design/ant-design/blob/master/package.json#L37-L39

```json
  "peerDependencies": {
    "react": ">=16.0.0",
    "react-dom": ">=16.0.0"
  }
```

Comment (bertho-zero) [reply src/Collapse.jsx]: And I also just noticed that there is no peerDependencies in rc-collapse, it's normal?

Comment (bertho-zero) [reply src/Collapse.jsx]: @afc163 I replaced `PureComponent` with `shouldComponentUpdate`.

**Comment by bertho-zero** [comment]
@afc163 Can you look at that?

**Pull request merged**